hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -114,6 +114,11 @@
       uvm_test_seq: chip_sw_base_vseq
       sw_test: sw/device/tests/dif/dif_plic_sanitytest
     }
+    {
```

Comment:
> I don't suppose you have run this test in DV to confirm it works

Could you please point me to instructions of how to run these tests?

> Would you like to have ALL tests designated as 'DIF sanity' run as a part of DV (I am fine with adding it, though this one is not particularly useful for achieving DV goals)?

I guess it only worth running this (and probably the rest) of sanity tests in CI alone? When I was doing this change, I simply added it to the same files as `dif_plic_sanitytest` (former `consecutive_irqs_test`). Is `dif_plic_sanitytest` useful for DV?

Reply:
Here's how you would run it:
```
./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.py -i chip_dif_uart_sanitytest --tool vcs
```
Use the `-i (-items)` switch to pick and choose the available tests you would like to run (same as the value of the `name` key in this hjson file).

You may need to talk to @imphil re. access to VCS installation / license at lowRISC. 

Unfortunately for the SW devs @ Google, getting access to VCS is a lot more tricky.

Reply:
`dif_plic_sanitytest` is actually not useful either since `chip_uart_tx_rx` already does a lot more than that. However, I am fine with keeping it since our simulation resource footprint is quite low. We will however not run it in the private CI (since it is hosted by lowRISC with limited # of VCS licenses), but we will run it in the nightly regressions (at Google).

Reply:
If it is not useful for DV, I will just remove this test.